cli/valet.php

```diff
@@ -561,6 +561,45 @@
         table(['Path', 'PHP Version'], $sites->all());
     })->descriptions('List all sites using isolated versions of PHP.');
 
+    /**
+     * Get the PHP executable path for a site.
+     */
+    $app->command('which-php [site]', function ($site) {
+        $host = Site::host($site ?: getcwd()).'.'.Configuration::read()['tld'];
+        $phpVersion = Site::customPhpVersion($host);
+
+        if (! $phpVersion) {
+            $path = getcwd().'/.valetphprc';
+            if (file_exists($path)) {
+                $phpVersion = trim(file_get_contents($path));
+            }
+        }
+
+        $phpVersion = $phpVersion ? PhpFpm::normalizePhpVersion($phpVersion) : null;
+
+        return output(Brew::getPhpExecutablePath($phpVersion));
```

Comment:
I don't think the ternary check is necessary here, because you already checked it for false-ness on line 571.

Also.. what if we run `normalizePhpVersion` at the top of `getExecutablePath`? Then A) you don't have to run it here or anywhere else, and B) you don't have to add my suggested line in the comments there

```suggestion
        return output(Brew::getPhpExecutablePath(PhpFpm::normalizePhpVersion($phpVersion)));
```